### PR TITLE
Real World Example for .. in Loops

### PR DESCRIPTION
**Pull request by cblant20**
I am not sold on how I am presenting the relevance of not knowing how long the line is. 
I am also not sold on using child as the thing we're iterating through because it doesn't quite map to properties of an object, but for each as a concept is not limited to objects. 

Happy to have feedback or change the example per your ideas! :)

### CODE REVIEW

**Comment by cblant20** [comment]
@barnes7td maybe @bdougie? maybe @mmaxwell ? Any feedback on this one?

File: courses/javascript-primer/07-loops/05-for-in-loop-i/body.md [hunk 1]
```diff
@@ -34,3 +34,28 @@ undefined
 ```
 
 Similar to a `for` loop's counter variable, a `for...in` loop requires that you declare a variable (`var key`) to hold the property referenced in each stage of the loop. This variable is followed by the `in` keyword, which is followed by the name of object that the loop will iterate through.
+
+
+### Real World Example
+
+You own a cotton candy food truck, and you've decided to set up at a local fair. 
+```var supplyOfCottonCandy = 1000;```
```
Comment (mmaxwell) [inline courses/javascript-primer/07-loops/05-for-in-loop-i/body.md]: I think for single lines, you can just use one back tick.

**Comment by mmaxwell** [comment]
I agree, that length doesn't matter. I also think this may be a bad example. When I think of a line, I think of an array because an array is an ordered list. You use an array when the order matters, and I view the line at a cotton candy truck as a FIFO line. The specs for `for...in` say that the keys are iterated in an arbitrary order, so the ordering is never guaranteed.

I'm not exactly sure how I would tweak this example, but I think that coupled with the term "length" would encourage someone to think about this in terms of an array instead of an object.

I'm about to hop into a meeting, but I will think about this and provide more thoughts later.

**Comment by cblant20** [comment]
@mmaxwell I appreciate this feedback. I'm going to close this PR because I think I need a different example here.

**Pull request closed without merging**